core/arch/arm/kernel/elf_load.c

```diff
@@ -38,6 +38,9 @@
 #include "elf32.h"
 #include "elf64.h"
 
+/* OP-TEE does not have SIZET_MAX */
+#define SIZET_MAX ((size_t)~((size_t)0))
```

Comment:
My fault, I changed the name from `SIZE_MAX` to `SIZET_MAX`. I'll do what you suggested.